test/check-application

```diff
@@ -27,37 +27,111 @@ class TestApplication(testlib.MachineCase):
+
+        # delete a stopped container
+        b.click('#containers-containers tbody tr:contains("alpine:latest") td.listing-ct-toggle')
+        b.wait_present('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        b.wait_present(".modal-dialog div")
+        b.wait_present(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        b.wait_not_present(".modal-dialog div")
+        self.pageReload()
+        self.loadAllContainerlist()
+        b.wait_present("#containers-containers")
+        b.wait_not_in_text("#containers-containers", "alpine:latest")
+
+        # delete a running container
+        b.wait_present("#containers-containers")
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        b.wait_present(".modal-dialog div")
+        b.wait_present(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        self.confirm()
+        b.wait_not_in_text("#containers-containers", "busybox:latest")
+
+    def confirm(self):
+        b = self.browser
+        b.wait_present(".modal-dialog div")
+        b.wait_present(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
 
+    def pageReload(self):
+        """Page reload and wait for it to be built up again"""
+        b = self.browser
+        b.reload()
+        b.enter_page("/podman")
+
+    def loadAllContainerlist(self):
+        """Load and expand the container list with all containers"""
+
+        b = self.browser
+        b.wait_present("#app")
+        b.wait_present(".content-filter div")
+        b.click("#containers-containers-filter button")
+        b.wait_present("#containers-containers")
+        # show all containers to interact with stopped ones
+        b.wait_present("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_present("#containers-containers")
+
+    def containerStopDropdown(self):
+        b = self.browser
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button span.caret')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button span.caret')
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr ul:contains("Stop")')
+        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr ul:contains("Restart")')
```

Comment:
This is just being used once, so IMHO better moved inline in the actual test, to make it easier to read.